### DISCUSSION TITLE
How can I get the Github student developer pack while still in school?

### DISCUSSION THREAD

**Discussion by jahongirdev** [community/community#84108] [GitHub Education] [answered]
### Hi everyone,

Hi guys!
I don't know how to get the Github Student Developer pack while still in school? My school does not appear in the search. And my school doesn't have any student id-card. Can anyone help?

**Comment by J1NX-82** [comment, marked as answer]
Hey there, follow these steps to get the Github Student Developer Pack
1)Visit the GitHub Education website: GitHub Education.
2)Sign in or create a GitHub account.
3)Go to the Student Developer Pack section and start the verification process.
4)If your school is not listed, you will be prompted to provide alternative documentation. 

Anything like an ID card would work.

Do mark the answer as accepted. Thanks♥

**Reply by sebastian-espeset** [reply]
Hey there J1nx! I tried this with an "Official enrollment" document, but gh didn't like it. Gh suggested that I "Please select proof type 'Other' for this image.", but I didn't see that as an option. Any pointers here?

**Reply by gsanlo** [reply]
i did this but it still saids it processing? does that mean it didnt go through all the way?

**Comment by TaimoorCodes** [comment]
I was experiencing the same issue earlier but I resolved it. Leaving this guide on Medium for anyone who can't seem to figure out how to get their GitHub Student Developer Pack.

https://medium.com/@taimooor/how-to-get-your-github-student-developer-pack-5cab9f600da9
